modules/rosa-sts-aws-requirements.adoc

```diff
@@ -0,0 +1,38 @@
+[id="rosa-sts-customer-requirements_{context}"]
+= Customer Requirements
+
+
+{product-title} (ROSA) clusters must meet several prerequisites before they can be deployed.
```

Comment:
I suggest changing this entire sentence to something that makes it clear that this applies only to STS deployments. Maybe something like: "The following prerequisites must be complete before you deploy ROSA clusters using STS."

Otherwise, this content will be confusing to users.

Reply:
Great catch - thanks!